base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by ``2π``, returning in the range ``\[0,2π)``.
```

Comment:
The `\` char isn't needed, sphinx seems to manage find without them. Also the ones further down, `\[0, y)` etc.